src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -43,6 +43,7 @@ public data class YamlConfiguration constructor(
     internal val sequenceStyle: SequenceStyle = SequenceStyle.Block,
     internal val singleLineStringStyle: SingleLineStringStyle = SingleLineStringStyle.DoubleQuoted,
     internal val multiLineStringStyle: MultiLineStringStyle = singleLineStringStyle.multiLineStringStyle,
+    internal val serializeComments: Boolean = false,
```

Comment:
Is there a use case you're thinking of where someone would want to disable this? (I'm wondering if we can just always serialize comments and reduce the complexity of having this be configurable.)

Reply:
Yeah, I think you're right, configuration option is redundant. Thanks for pointing it out! And sorry, it's my first actual PR, so I'm making some mistakes on the way

Reply:
No worries at all, congratulations on your first PR 🎉